documentation/docs/index.md

```diff
@@ -1,12 +1,67 @@
-# Project "Piper" - extended library documentation
+# Project "Piper" User Documentation
 
-On the following pages, you can find documentation for all steps and scripts
-contained in the library of project "Piper".
+An efficient software development process is vital for success in building
+business applications on SAP Cloud Platform or SAP on-premise platforms. SAP
+addresses this need for efficiency with project "Piper". The goal of project
+"Piper" is to substantially ease setting up continuous delivery processes for
+the most important SAP technologies by means of Jenkins pipelines.
 
-We have also compiled a list of plugins that are required for the library to
-work.
+## What you get
 
-For information on the project and installation of the library, please have a
-look at our [README.md][piper-pipelines-readme].
+Project "Piper" consists of two parts:
 
-[piper-pipelines-readme]: https://github.com/SAP/jenkins-library/blob/master/README.md
+* [A shared library][piper-library] containing steps and utilities that are
+  required by Jenkins pipelines.
+* A set of [Docker images][devops-docker-images] used in the piper library to implement best practices.
+
+The shared library contains all the necessary steps to run our best practice
+[Jenkins pipelines][piper-library-pages] described in the Scenarios section or
+to run a [pipeline as step][piper-library-scenario].
+
+The best practice pipelines are based on the general concepts of [Jenkins 2.0
+Pipelines as Code][jenkins-doc-pipelines].  With that you have the power of the
+Jenkins community at hand to optimize your pipelines.
+
+You can run the best practice Jenkins pipelines out of the box, take them as a
+starting point for project-specific adaptations or implement your own pipelines
+from scratch using the shared library.
+
+## Extensibility
+
+If you consider adding additional capabilities to your `Jenkinsfile`, consult
+the [Jenkins Pipeline Steps Reference][jenkins-doc-steps]. There, you get an
+overview about steps that are natively supported by Jenkins.
+
+The [Jenkins shared libraries][jenkins-doc-libraries] concept helps you to
+extract reusable parts from your pipeline and to keep your pipeline code small
+and easy to maintain.
+
+Custom library steps can be added using a custom library according to the
+[Jenkins shared libraries][jenkins-doc-libraries] concept instead of adding
+groovy coding to the `Jenkinsfile`. Your custom library can coexist next to the
+provided pipeline library.
+
+## API
+
+All steps (`vars` and `resources` directory) are intended to be used by Pipelines and are considered API.
```

Comment:
The steps are only contained in `vars`. To my understanding `resources` is not an API.

Reply:
out of scope, already existing,